packages/core/src/conflict-resolution.ts

```diff
@@ -274,7 +197,7 @@ export function commitAtTime(stateHolder: StreamStateHolder, timestamp: number):
 export class ConflictResolution {
   constructor(
     public logger: DiagnosticsLogger,
-    public anchorValidator: AnchorValidator,
+    public anchorTimestampExtractor: AnchorTimestampExtractor,
```

Comment:
Could we make it private please? It's easier to avoid circular dependencies if we have clear module boundaries and modules can't access instances attached to other modules.

Reply:
done